higlass/trackdata.py

```diff
@@ -0,0 +1,76 @@
+import slugid
+from higlass.client import Track
+from pathlib import Path
+from typing import List, Tuple, Union
+from numpy import cumsum
+
+import itertools as it
+from pathlib import Path
+
+def chromsize_pairs(chromsizes_fn):
+    with open(chromsizes_fn, 'r') as f:
+        chroms = [l.strip().split('\t') for l in f.readlines()]
+        chromnames = [c[0] for c in chroms]
+        chromlengths = [int(c[1]) for c in chroms]
+    
+    return list(zip(chromnames, chromlengths))
+
+def load_chromsizes(chrom_names_lengths):
+    chromnames, chromlengths = zip(*chrom_names_lengths)
+
+    cumlengths = list(it.accumulate(chromlengths))
+    # we want the cumlengths to start 0
+    cumlengths = [0] + cumlengths[:-1]
+
+    return dict([(name, {
+        'name': name,
+        'length': length,
+        'start': start
+    }) for name, length, start in zip(chromnames, chromlengths, cumlengths)])
+
+def bedtiles(lines: List[Tuple[str, int, int, str, str, str]],
```

Comment:
What is the difference between `trackdata` and tilesets?

Reply:
Trackdata does not require a server. It uses [local tiles whose values are encoded in the viewconf](https://docs.higlass.io/view_config.html#raw-tile-values).

Reply:
I added a note to the docs briefly explaining this.

Reply:
This is all just semantics but I find the naming confusing. Why not call it _local_ or _inline_ data? Especially in the context of the Track object where it's also just referred to as `data=data`. I assume `data` is inspired by the viewconf prop with the same name but I wonder how should a new user know that the `data` property is meant for a very special use case only?

I would propose to rename `data` to `local_tiles`, `trackdata` to `inline_data` or something similar.

Anyway, just my 2 cents. Not critical

Reply:
I agree with @flekschas's naming suggestions. Maybe even call both of them `inline_tiles`.

Reply:
I'm on board with naming `trackdata` to `inline_data`. I'm a little hesitant to name the `Track` parameter `inline_tiles` because it can be used for other types of data (e.g. `divided` datasets or BAM file urls).